acados/ocp_nlp/ocp_nlp_common.h

```diff
@@ -218,11 +218,15 @@ ocp_nlp_in *ocp_nlp_in_assign(ocp_nlp_config *config, ocp_nlp_dims *dims, void *
 
 typedef struct
 {
-    struct blasfeo_dvec *ux;
-    struct blasfeo_dvec *z;
-    struct blasfeo_dvec *pi;
-    struct blasfeo_dvec *lam;
-    struct blasfeo_dvec *t;  // slacks of inequalities
+    struct blasfeo_dvec *ux;  // NOTE: this contains [u; x; s_l; s_u]! - rename to uxs?
+    struct blasfeo_dvec *z;  // algebraic vairables
+    struct blasfeo_dvec *pi;  // multipliers for dynamics
+    struct blasfeo_dvec *lam;  // inequality mulitpliers
+    struct blasfeo_dvec *t;  // variables corresponding to
+       // the evalution of the inequalities (at the solution)
```

Comment:
what about "slack variables associated with (all) inequality constraints (and not necessarily only with soft constraints)"

Reply:
For soft constraints `t` and `s` still have different values, so I wanted to make sure the meaning is mentioned somehow.
I readded the word slack though (before reading your comment here)